ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/sync/fullsync/FullImportBlockStep.java

```diff
@@ -56,12 +60,28 @@ public void accept(final Block block) {
     if (!importer.importBlock(protocolContext, block, HeaderValidationMode.SKIP_DETACHED)) {
       throw new InvalidBlockException("Failed to import block", blockNumber, block.getHash());
     }
+    gasAccumulator += block.getHeader().getGasUsed();
     int peerCount = -1; // ethContext is not available in tests
     if (ethContext != null && ethContext.getEthPeers().peerCount() >= 0) {
       peerCount = ethContext.getEthPeers().peerCount();
     }
     if (blockNumber % 200 == 0) {
-      LOG.info("Import reached block {} ({}), Peers: {}", blockNumber, shortHash, peerCount);
+      final long nowMilli = Instant.now().toEpochMilli();
+      if (lastReportMillis == 0 || gasAccumulator == 0) {
+        // this is a formatted logger statement
+        //noinspection PlaceholderCountMatchesArgumentCount
+        LOG.info("Import reached block %d (%s), Peers: %d", blockNumber, shortHash, peerCount);
```

Comment:
Why not just allow it to log the 0? It doesn't do much harm and it makes it more consistent, grepable, etc.

Reply:
updated it to log a `-` when either time is not accurate or gas burnt is zero.  Greps consistently now.